index.js

```diff
@@ -31,6 +31,7 @@
 
 import { MediaPlayer } from './index_mediaplayerOnly';
 
+import { supportsMediaSource as isSupported } from './src/streaming/utils/Capabilities'
```

Comment:
I prefer to no renaming here as we might expose additional functions from the Capabilites (or the whole Capabilities) class at some point.